### PR TITLE
onioncat 0.2.2.r569 (new formula)

### PR DESCRIPTION
**Pull request by myfreeweb**

### CODE REVIEW

File: Library/Formula/onioncat.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class Onioncat < Formula
+  desc "A VPN-adapter that provides location privacy using Tor or I2P"
```
Comment (bfontaine) [inline Library/Formula/onioncat.rb]: Please remove the indefinite article at the beginning: `desc "VPN-adapter ...`

**Comment by DomT4** [comment]
Should this maybe have a `tor` recommended dependency, or?

**Comment by myfreeweb** [comment]
> `:recommended` generates an implicit `without-foo` option, meaning that the dependency is enabled by default and the user must pass `--without-foo` to disable this dependency. The default description can be overridden using the normal option syntax (in this case, the option declaration must precede the dependency)

Is there a way to recommend a dependency that's, like, runtime-optional and doesn't affect builds?

**Comment by DomT4** [comment]
`depends_on "tor" => [:recommended, :run]` might do the job.

**Comment by myfreeweb** [comment]
It still adds a `--without-tor` option which doesn't make sense.

**Comment by bfontaine** [comment]
You can use `:optional` instead of `:recommended` for the opposite behavior: it doesn’t install it unless you pass it `--with-tor`.

**Comment by myfreeweb** [comment]
Oh. Wait. The `--without-tor` option is for Homebrew to not require Tor, right?

*Somehow* I thought it gets passed to `configure` :D Obviously, `configure` options are fully specified *in* the formula, not added automatically.

**Comment by bfontaine** [comment]
> Oh. Wait. The `--without-tor` option is for Homebrew to not require Tor, right?
>
> *Somehow* I thought it gets passed to `configure` :D Obviously, `configure` options are fully specified *in* the formula, not added automatically.

Yep.

File: Library/Formula/onioncat.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+class Onioncat < Formula
+  desc "VPN-adapter that provides location privacy using Tor or I2P"
+  homepage "https://www.onioncat.org"
+  url "https://www.cypherpunk.at/ocat/download/Source/current/onioncat-0.2.2.r569.tar.gz"
```
Comment (DomT4) [inline Library/Formula/onioncat.rb]: Would you mind setting an explicit `version` here? It's currently reading as `569`. After that I'm happy to merge this.

```ruby
url "https://www.cypherpunk.at/ocat/download/Source/current/onioncat-0.2.2.r569.tar.gz"
version "0.2.2.r569"
```

Comment (DomT4) [inline Library/Formula/onioncat.rb]: Ping on this?

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/16456b79f7dfb216bd156c3da79fc3eb5d882fa9. Thank you for your contribution to Homebrew @myfreeweb; we appreciate it! :smiley_cat:

**Pull request closed without merging**